schemas/common_definitions.json

```
@@ -96,6 +96,32 @@
     },
     "required": ["when"]
   },
+  "new_skip_conditions": {
+    "description": "Allows an element to be skipped when a condition has been met. By adding more than one `when` element it will evaluate as an or rule.",
+    "type": "object",
+    "minItems": 1,
+    "uniqueItems": true,
+    "items": {
+      "type": "object",
+      "properties": {
+        "when": {
+          "$ref": "https://eq.ons.gov.uk/rules/rule.json"
+        }
+      },
+      "additionalProperties": false
+    },
+    "required": ["when"]
```

"[items](https://json-schema.org/understanding-json-schema/reference/array.html#items)" are used for arrays, our new struct is an object, therefore, this would result in some incorrect validation, for example, this defenition would pass validation as it isn't able to check the rule properly:.
```
  "skip_conditions": {
    "when": {
      "==": [

      ]
    }
  }
```

You need to remove the items check and move the proprieties up a level.

```suggestion
    "properties": {
      "when": {
        "$ref": "https://eq.ons.gov.uk/rules/rule.json"
      }
    },
    "additionalProperties": false,
    "required": ["when"]
```